src/stokes/Stokes3D.jl

```diff
@@ -346,15 +302,21 @@ function JustRelax.solve!(
             @parallel (@idx ni) compute_ρg!(ρg[3], rheology, args)
 
             ν = 1e-3
```

Comment:
is the damping factor a remnant here? as you assign 1.0 in the function itself. 
Same thing at L491.

Reply:
1.0 is for testing the plasticity on this particular benchmark, it will use 1e-2 or 1e-3 once merged